pyramid_oereb/contrib/sources/interlis_2_3/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+This package offers access to the standard database structure. All defined sources are belonging directly to
```

Comment:
Maybe call it not standard database structure since this is the pyramid_oereb own already. Better call it transfer structure.